test/acceptance/duplicate_tx_test.go

```diff
@@ -0,0 +1,85 @@
+package acceptance
+
+import (
+	"bytes"
+	"context"
+	"github.com/orbs-network/orbs-network-go/test/builders"
+	"github.com/orbs-network/orbs-network-go/test/harness"
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestSendSameTransactionMoreThanOnce1(t *testing.T) {
```

Comment:
can you find better names than `1` and `2`?

Reply:
done